src/loader.js

```diff
@@ -18,11 +18,11 @@ module.exports = function (content) {
     context: getContext(this),
     content: msgpack.encode([content, query.options, pkg.version]), // hash must depend on logo + config + version
   }));
-
+  const outputPath = query.outputPath ? trailingSlash(query.outputPath) : null;
```

Comment:
I believe the intention is to have `outputPath` fallback to `prefix` instead of `null`.